### PR TITLE
feat: add react router and create main routes.

### PR DESCRIPTION
**Pull request by yomar-dev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/react-router-dom": "^5.1.5",
```
Comment (dnrgdev) [inline package.json:6]: Move tho `devDependencies`

File: src/index.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-import App from 'App';
 import * as serviceWorker from 'serviceWorker';
 
 import 'styles/styles.scss';
+import { AppRouter } from 'routes/AppRouter';
 
 ReactDOM.render(
   <StrictMode>
-    <App />
+    <AppRouter />
```
Comment (dnrgdev) [inline src/index.tsx:10]: i don't know if AppRouter is good because they contains more that router match

Comment (yomar-dev) [reply src/index.tsx:10]: No worries, this is an example. Later when everything change the Navbar will be move to his corresponding component.

Comment (dnrgdev) [reply src/index.tsx:10]: ok

**Pull request merged**